packages/dd-trace/src/format.js

```diff
@@ -96,6 +100,18 @@ function extractTags (trace, span) {
   addTag(trace.meta, trace.metrics, HOSTNAME_KEY, hostname)
 }
 
+function extractRootTags (trace, span) {
+  const context = span.context()
+  const isLocalRoot = span === context._trace.started[0]
+  const parentId = context._parentId
+
+  if (!isLocalRoot || (parentId && parentId.toString(10) !== '0')) return
```

Comment:
Rather than doing a `toString`, can we not do a comparison w/ a/the zero id?

Reply:
No because it's not used in all cases.